src/amber/cli/templates/app/spec/spec_helper.cr

```diff
@@ -11,9 +11,6 @@ Micrate::DB.connection_url = Amber.settings.database_url
 # Automatically run migrations on the test database
 Micrate::Cli.run_up
 
-# Disable query logger for tests
-Granite.settings.logger = Logger.new nil
-
```

Comment:
Maybe we can set it dynamically? Take a look [here](https://stackoverflow.com/a/45908692/1586762)

Reply:
@veelenga yeah, but I think Granite logs are very useful, you can see query time and raw sql information, these logs helped me to fix some issues on Amber itself :sweat_smile:

Reply:
Okay, even better 👍

Reply:
I'd prefer to leave it disabled because it generates so much noise. A verbose query log makes it hard to tell what happens when a test is failing. When tests all pass, the output of the suite should be minimal, indicating that nothing unexpected happened.

Reply:
@robacarp Crecto spec fails because granite dependency is missing on crecto projects. Maybe we should use a `spec_helper.cr.ecr` template as well, like  this:

```ecr
<%- if database == "granite" %>
Granite.settings.logger = Logger.new nil
<% end %>
```

WDYT?

Reply:
@faustinoaq yes, thats a good idea.

Reply:
@faustinoaq what happened here? Did the idea for the conditional get scrapped?

Reply:
@robacarp Yeah, let me add this conditional code :+1: